### PR TITLE
Add acct_nt (T2 Lock)

### LINKED ISSUE #461: acct_nt is not on the wiki
### Context

Add acct_nt to the wiki.

### PR DESCRIPTION
**Pull request by MichaelEpicA**
Adds a full description and explanation of every mode and every way acct_nt can behave.

### Problem

Fixes #461

### CODE REVIEW

File: acct_nt.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Find the deposit that acct_nt asks for and provide it as the acct_nt argument. It may be a bit off from the timestamp.
+
+</details>
+
+### Mode: net
+
+<details>
+<summary>Spoilers for acct_nt's solutions </summary>
+
+Look at the timestamps that acct_nt gives you, and add up all the transactions from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.
```
Comment (Zachava96) [inline acct_nt.mdx:69]: All articles should be written in third person with no referential nouns (e.g. you, your), except for Tutorials, FAQs, and Writeups.

```suggestion
Add up all the transactions between the two timestamps, with incoming transactions being positive and outgoing transactions being negative. The actual starting and ending timestamps may be incorrect by up to one minute.
```

File: acct_nt.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+</details>
+
+### Mode: total earned
+
+<details>
+<summary>Spoilers for acct_nt's solutions </summary>
+
+Look at the timestamps that acct_nt gives you, and add up all the transactions where you gained money from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.
```
Comment (Zachava96) [inline acct_nt.mdx:78]:
```suggestion
Add up all the incoming transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
```

File: acct_nt.mdx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+</details>
+
+### Mode: total spent
+
+<details>
+<summary>Spoilers for acct_nt's solutions </summary>
+
+Look at the timestamps that acct_nt gives you, and add up all the transactions where you lost money from the starting timestamp to the end. The actual starting and ending timestamps may be a bit off from what acct_nt gives you.
```
Comment (Zachava96) [inline acct_nt.mdx:87]:
```suggestion
Add up all the outgoing transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
```

File: acct_nt.mdx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+</details>
+
+### Mode: with/without memos
+
+<details>
+<summary>Spoilers for acct_nt's solutions </summary>
+
+With or without memos indicates whether or not if you should consider transactions with memos. If it says with memos, make sure the transactions have memos before adding them, otherwise ignore them.
```
Comment (Zachava96) [inline acct_nt.mdx:96]:
```suggestion
Indicates whether only transactions with (or without) memos should be considered.
```

File: acct_nt.mdx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Memos are a little message that can be sent with a transaction. It displays as memo: "message here" in [[accts.transactions]].
```
Comment (Zachava96) [inline acct_nt.mdx:97]: I don't think that memos need to be explained here; if needed, they can be explained on the accts.transactions page.
```suggestion
```

**Review by Zachava96 — CHANGES_REQUESTED** [review]
All articles should be written in third person with no referential nouns (e.g. you, your), except for Tutorials, FAQs, and Writeups.

**Review by Zachava96 — APPROVED** [review]

**Review by Zachava96 — CHANGES_REQUESTED** [review]
Total spent/earned actually uses net, not just incoming or outgoing, my bad

File: acct_nt.mdx [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+Add up all the incoming transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
+
+</details>
+
+### Mode: total spent
+
+<details>
+<summary>Spoilers for acct_nt's solutions </summary>
+
+Add up all the outgoing transactions between the two timestamps. The actual starting and ending timestamps may be incorrect by up to one minute.
```
[the remark this answers is not in the document]
Comment (MichaelEpicA) [reply acct_nt.mdx:87]: Are you sure they use net? That just seems like a redundant thing then, and it should probably just be put under one category.

**Comment by hmdunce** [comment]
Thanks both of you! I was going to see about merging but looks like there is still discussion.

I think it doesn't have to be perfect. Someone can always fix it later.

**Review by hmdunce — CHANGES_REQUESTED** [review]
The file should be moved to docs/upgrades/locks so that it shows up in the locks list. Currently the page doesn't show up at all I think.

You can do it with
```
git mv acct_nt.mdx docs/upgrades/locks/
```
and then commit, push as usual.

I would just do this but I don't want to cause confusion for ongoing work on the file.

**Review by hmdunce — APPROVED** [review]
OK, I checked in codespaces and it looks good to me.

I am not checking for 100% accuracy since I don't think I've ever fully understood how acct_nt works, but I think that's fine.

I only have one question: do the stats and behavior sections contain spoilers? I've seen the README definition but I'm not sure exactly what it means. :smile:

I don't know how much it matters though.

**Comment by hmdunce** [comment]
The meaning of `acct_nt_min` in particular does not seem obvious to me. I suspect what is written here may not even be correct??? :man_shrugging:

**Comment by MichaelEpicA** [comment]
acct_nt_min as close to my understanding is correct, since acct_nt uses a range of transactions and acct_nt_min is how small that range can be

**Comment by hmdunce** [comment]
Just to be safe, I will add a commit, you can tell me if it looks good to you?

By the way, just curious, are you able to push the "Squash and merge" button when it turns green? I don't know who has permission. If you are able, you should feel free to push it, IMO.

**Comment by MichaelEpicA** [comment]
I do not have write access so no

**Comment by hmdunce** [comment]
Ok, it was probably unnecessary, but added some spoiler guards. If it looks good to you I will merge?

Thanks for your contribution!

**Comment by MichaelEpicA** [comment]
Yep

**Pull request merged**